### PR TITLE
back button

### PR DESCRIPTION
**Pull request by sean-seibel**
added back button in top right of leaderboard page

### CODE REVIEW

**Review by sean-seibel — COMMENTED** [review]
Screen Shot 2023-01-12 at 9 50 11 PM

File: components/stack/FullStack.tsx [hunk 1]
```diff
@@ -100,11 +100,16 @@ export const Leaderboard = () => {
   if (isLoading) return <>Loading...</>;
   if (data) {
     return (
-      <div>
-        {data.map((place: LeaderboardPlace, index: number) => (
-          <LeaderboardRow key={index} place={place} num={index + 1} />
-        ))}
-      </div>
+      <>
```
Comment (lpfahler16) [inline components/stack/FullStack.tsx]: Change this to `<div className="relative">`. This will make the positioning relative to this parent

**Review by lpfahler16 — CHANGES_REQUESTED** [review]
Move the changes to the leaderboard page.

**Comment by sean-seibel** [comment]
Screen Shot 2023-01-12 at 10 17 02 PM

**Comment by sean-seibel** [comment]
Screen Shot 2023-01-12 at 10 27 07 PM

**Comment by sean-seibel** [comment]
Screen Shot 2023-01-12 at 10 43 44 PM

**Comment by sean-seibel** [comment]
Screen Shot 2023-01-12 at 10 45 53 PM

**Pull request merged**